File: src/classes/api/req_res/update-idf-kyou-response.ts
'use strict'

import { IDFKyou } from '@/classes/datas/idf-kyou'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateIDFKyouResponse extends GkillAPIResponse {

    updated_idf_kyou: IDFKyou

    updated_idf_kyou_kyou: Kyou

    constructor() {
        super()
        this.updated_idf_kyou = new IDFKyou()
        this.updated_idf_kyou_kyou = new Kyou()
    }

}

File: src/app/gkill/api/req_res/update_idf_kyou_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type UpdateIDFKyouResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedIDFKyou *reps.IDFKyou `json:"updated_idf_kyou"`

	UpdatedIDFKyouKyou *reps.Kyou `json:"updated_idf_kyou_kyou"`
}
